### PR TITLE
Courses index view

### PR DESCRIPTION
**Pull request by MatiasMercado**
## Courses Index View

- Nothing but the view for the courses =)

## Trello Card
https://trello.com/c/timMxqFn/48-maquetar-vista-de-cursos

### CODE REVIEW

File: app/views/courses/index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<div ng-controller='CoursesIndexCtrl as controller'>
+  <div class='courses-index-container'>
+    <!-- Page Heading -->
```
Comment (MatiasComercio) [inline app/views/courses/index.html]: Copy the student's index page heading style

File: app/views/courses/index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          <tr ng-hide='controller.courses.length > 0' class='bg-warning'>
+            <td colspan='4' class='text-danger text-center' translate='i18nNoCoursesFound'></td>
+          </tr>
+          <tr ng-repeat='course in controller.courses'>
+            <td>{{ course.courseId }}</td>
+            <td>{{ course.name }}</td>
+            <td>{{ course.credits }}</td>
+            <td>{{ course.semester }}</td>
+            <td class='actions-container'>
+              <a class='btn btn-default full-width-button' role='button'>
```
Comment (MatiasComercio) [inline app/views/courses/index.html]: Remove the `full-width-button` class. This is now accomplished through flexbox.

File: app/views/courses/index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <!-- Content -->
+    <div class='table-responsive'>
+      <table class='table table-hover'>
+        <thead>
+          <tr>
+            <th translate='i18nCourseId'></th>
+            <th translate='i18nCourseName'></th>
+            <th translate='i18nCredits'></th>
+            <th translate='i18nSemester'></th>
+            <th translate='i18nActions'></th>
```
Comment (MatiasComercio) [inline app/views/courses/index.html]: This translation is not working OK. Check that the translation is' i18**n**Actions' at the translations file

File: app/views/courses/index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          </tr>
+        </thead>
+        <tbody>
+          <tr ng-hide='controller.courses.length > 0' class='bg-warning'>
+            <td colspan='4' class='text-danger text-center' translate='i18nNoCoursesFound'></td>
```
Comment (MatiasComercio) [inline app/views/courses/index.html]: Check how it is seen the message `No se encontraron materias disponibles` is seen. After that, update this to be `colspan='5'`. This is telling how many table columns the `td` element should be using.

File: app/views/courses/index.html (hunk 3 above)
Comment (MatiasMercado) [inline app/views/courses/index.html]: Thx, I may have missed this during the rebase

**Comment by MatiasComercio** [comment]
Remove your assignment and assign me when finished with those fixes

**Pull request merged**